modules/tinymce/src/plugins/fullscreen/test/ts/browser/FullScreenPluginTest.ts

```diff
@@ -89,9 +113,12 @@ describe('browser.tinymce.plugins.fullscreen.FullScreenPluginTest', () => {
         plugins: 'fullscreen link',
         base_url: '/project/tinymce/js/tinymce',
         setup: (editor: Editor) => {
-          lastEventArgs.set(null);
-          editor.on('FullscreenStateChanged', (e: Editor) => {
-            lastEventArgs.set(e);
+          firedEvents = [];
+          editor.on('FullscreenStateChanged ResizeEditor', (e: any) => {
+            firedEvents.push(e.type);
+            if (Type.isBoolean(e.state)) {
+              firedEvents.push(e.state);
+            }
```

Comment:
I think this would be a lot better if we refactor it to all be strings. So make them be [`'fullscreenstatechanged:true', 'resizeeditor'`] or something that would also make the assert if failing be a lot more clear of what was wrong. I really dislike having multiple different types on things. Even if TS allows that other languages that are typed generally don't it also simplifies reasoning and avoids type guards etc on other parts of the code, in this example it not a big problem but it's the general idea of mixing types that should be avoided if possible.

```suggestion
          editor.on('FullscreenStateChanged ResizeEditor', (e: any) => {
            if (Type.isBoolean(e.state)) {
              firedEvents.push(`${e.type}:${e.state}`);
            } else {
              firedEvents.push(e.type);
            |
```